netlify.toml

```
@@ -1,3 +1,201 @@
 [build]
   publish = "dist"
   command = "npm run build-docs"
+
+# redirects for older pages:
+# Fundementals
+[[redirects]]
+  from = "/fundementals/principles*"
```

Typo: should be fund**a**mentals, here and below.

```suggestion
  from = "/fundamentals/principles*"
```